src/Okta.DNX.OAuth.ResourceServer/Project_Readme.html

```diff
@@ -0,0 +1,187 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Welcome to ASP.NET Core</title>
+    <style>
+        html {
+            background: #f1f1f1;
+            height: 100%;
+        }
+
+        body {
+            background: #fff;
+            color: #505050;
+            font: 14px 'Segoe UI', tahoma, arial, helvetica, sans-serif;
+            margin: 1%;
+            min-height: 95.5%;
+            border: 1px solid silver;
+            position: relative;
+        }
+
+        #header {
+            padding: 0;
+        }
+
+            #header h1 {
```

Comment:
nit: spacing. I can see what you're doing here (indenting because it's nested), but generally we don't do that.